examples/computer_vision/mnist_pl/data.py

```diff
@@ -1,3 +1,31 @@
+# Copyright Determined AI. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# Copyright The PyTorch Lightning team.
```

Comment:
Please leave a comment above this header that explains in plain English which parts of the code are copied or derived from PTL.  Technically, PTL does not hold a copyright on the changes we make to their code; they only own copyrights to the original code.

One of the licensing requirements of that original code is that this header goes where their code goes.  By using their code, we are effectively licensing their code, so we have to comply with this requirement.

But we definitely should clearly indicate what parts of the code actually belong to them.

Here's an example from yogadl:

```python
#
# `normalize_img()` and the model definition are derived from the TensorFlow
# documentation: https://www.tensorflow.org/datasets/keras_example
#
# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```